Headers/AppKit/NSUserInterfaceItemIdentification.h

```diff
@@ -1,44 +1,56 @@
-/** NSUserInterfaceItemIdentification.h
-   <abstract>Associate a unique identifier with objects in your user interface</abstract>
-
-   Copyright <copy>(C) 2017 Free Software Foundation, Inc.</copy>
-
-   Author: Daniel Ferreira <dtf@stanford.edu>
-   Date: 2017
-
-   This file is part of the GNUstep GUI Library.
-
+/* Definition of class NSUserInterfaceItemIdentification
+   Copyright (C) 2020 Free Software Foundation, Inc.
+   
+   By: Gregory John Casamento
+   Date: Tue Apr 14 13:46:36 EDT 2020
+
+   This file is part of the GNUstep Library.
+   
    This library is free software; you can redistribute it and/or
    modify it under the terms of the GNU Lesser General Public
    License as published by the Free Software Foundation; either
-   version 2 of the License, or (at your option) any later version.
-
+   version 2.1 of the License, or (at your option) any later version.
+   
    This library is distributed in the hope that it will be useful,
    but WITHOUT ANY WARRANTY; without even the implied warranty of
-   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.	 See the GNU
-   Lesser General Public License for more details.
-
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+   Library General Public License for more details.
```

Comment:
LIBRARY????

Reply:
Ugh... the script.  No worries, I'll fix it.